Build/Build.cs

```diff
@@ -351,36 +355,110 @@ void ReportTestOutcome(params string[] globFilters)
 
     Target SpellCheck => _ => _
         .OnlyWhenDynamic(() => RunAllTargets || HasDocumentationChanges)
+        .DependsOn(InstallNodeWithNpm)
         .ProceedAfterFailure()
         .Executes(() =>
         {
-            Node($"{YarnCli} --silent install", workingDirectory: RootDirectory,
-                logger: YarnInstallLogger);
-
-            Node($"{YarnCli} --silent run cspell", workingDirectory: RootDirectory,
+            Npm($"install --silent", workingDirectory: RootDirectory);
+            Npm($"run --silent cspell", environmentVariables: NpmEnvironmentVariables,
```

Comment:
Maybe we should remove the silent flag. I'm not even sure if it is doing anything.

Reply:
`--silent` is actually doing something
See:

| `npm install` | `npm run` |
|--------|--------|
| ![image](https://github.com/fluentassertions/fluentassertions/assets/44049228/4f17d6a8-3fa7-4981-9169-089063c04a4f) |  ![image](https://github.com/fluentassertions/fluentassertions/assets/44049228/b3f8ab3e-c3ed-4ee4-8f93-701bc52e2de7)|

Reply:
Hmm, maybe we should keep the left view. But I don't have a strong opinion about that.

Reply:
ähm.. the left view? Remove `npm run cspell`? :joy:

the left is both `npm install`; the right both `npm run` (with and without `--silent`

Reply:
Oh yes, you're right. Never mind me.

Reply:
No problem ;)

Reply:
Resolved?